src/appsco-app.html

```diff
@@ -4642,6 +4680,47 @@
                 this._addCustomer(customer);
                 this._notify('New customer ' + customer.name + ' was successfully added.');
             },
+
+            _onRemoveCustomersAction: function() {
+                var selectedCustomers = this.$.appscoCustomersPage.getSelectedCustomers();
+
+                if (selectedCustomers.length > 0) {
+                    this.set('_selectedCustomers', selectedCustomers);
+                    this.$.appscoCustomersRemove.toggle();
+                }
+                else {
+                    this._onHideCustomersBulkActions()
+                }
+            },
+
+            _onRemovedCustomers: function() {
+                $customersCount = this.$.appscoCustomersPage.removeCustomers(this._selectedCustomers);
```

Comment:
Should be 'var customersCount', not PHP :) . And why customersCount? removeCustomers method returns number of removed customers?